pkg/agent/util/net_others.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2021 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build darwin
+
+package util
+
+import (
+	"fmt"
+	"net"
+)
+
+// GetVirtualInterfaceByName is not implemented. An error returned when this function is called, which means the required
+// interface is not found.
+func GetVirtualInterfaceByName(ifaceName, ovsBridgeName string) (*net.Interface, error) {
```

Comment:
I understand this cause compiliation issue on MAC but this is not the only missing function, can we rename net_linux to net_other instead?